server/routerlicious/packages/routerlicious/config/config.json

```diff
@@ -223,6 +223,7 @@
         "serverUrl": "http://localhost:3003",
         "blobStorageUrl": "http://historian:3000",
         "internalBlobStorageUrl": "http://historian:3000",
+        "gitrestUrl": "http://gitrest:3000",
```

Comment:
Instead of adding a `gitrestUrl` to `worker`, can we follow Zach's suggestion in here (https://github.com/microsoft/FluidFramework/pull/9849#discussion_r848874932) and do it in the `storage` section? Do you think it would make more sense?

First, I think `storageUrl` is a better name than `gitrestUrl`, since the storage implementation might be something other than `gitrest`.

Second, even though we already had this `worker` section, I think it makes more sense to add the `storageUrl` to the `storage` section in the config. That's where we control storage-related things like enabling whole summary upload.

Reply:
Got it. Moved to the storage section as the storageUrl. :)